backend/deno.json

```diff
@@ -1,4 +1,14 @@
 {
+  "importMap": "../import_map.json",
+  "tasks": {
+    "start": "deno run     --no-check=remote --allow-read=. --allow-net=localhost --allow-env --watch src/app.ts",
+    "build": "deno compile --no-check=remote --allow-read=. --allow-net=localhost --allow-env -o dist/jimmi src/app.ts",
+    "lint": "deno lint",
+    "fmt": "deno fmt",
+    "fmt:check": "deno fmt --check",
```

Comment:
@Music-Bot-for-Jitsi what do you think, should we keep the scripts currently invoked by e.g. `deno task lint` or should we drop the lint, fmt and fmt:check as it is shorter to simply type `deno lint`. The result is the same as the config file is recognized automatically.

Reply:
I think it is cleaner if we use `deno task [...]`. Makes it more consistent and let us know if it is a command defined by Deno or not.